scene/3d/camera_3d.cpp

```diff
@@ -602,7 +602,7 @@ void Camera3D::set_fov(real_t p_fov) {
 }
 
 void Camera3D::set_size(real_t p_size) {
-	ERR_FAIL_COND(p_size < 0.001 || p_size > 16384);
+	ERR_FAIL_COND(p_size <= CMP_EPSILON);
```

Comment:
Note that changing `0.001` to `CMP_EPSILON` makes it possible to set the size to 0.0001 with `set_size()` but not via the inspector. Probably not much of a problem though, since such a value would hardly ever be used.

Reply:
That's the point, Camera3D's allowed values should be looser than the inspector's allowed values.

Reply:
That'll make the inspector display a wrong value when using the remote tree. e.g. when the size is set to 0.0001 by code, the inspector will say it's 0.001.

Reply:
That seems like a separate problem then, we should make the remote inspector ignore all range hints so that it always displays the real value. I can see this being an issue in other circumstances too.